luigi/interface.py

```diff
@@ -238,8 +238,6 @@ def parse_task(self, cmdline_args=None):
 
         _add_global_parameters(parser)
 
-        task_names = Register.task_names()
```

Comment:
Did flake8 really find this??? How does it know the side effects are not needed?